susemanager-utils/testing/docker/master/uyuni-push-to-obs/add_repositories.sh

```diff
@@ -2,3 +2,4 @@
 set -e
 
 zypper ar -f https://download.opensuse.org/repositories/home:/mcalmer:/tito/openSUSE_Leap_15.0/home:mcalmer:tito.repo
+zypper ar -f http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_15.0/SUSE:CA.repo
```

Comment:
hmm, this is an internal repo. So the container cannot be build without access to the SUSE internal network.
Maybe it would be better to get the RPM and put it in the git?

Reply:
@mcalmer I wonder if there are restrictions to redistribution.

Let's merge the PR, so we can get this working by now, and will check with... Ciaran? Autobuild?

Maybe we can just make it available at OBS.

Reply:
fine with me